### PR TITLE
Remove [interactive] pip install option

### LINKED ISSUE #508: explore_data tutorial instrument_view with issue
Hi all,

Was trying out `scippneutron` and following up with the tutorial [1_exploring-data.html](https://scipp.github.io/scippneutron/tutorials/1_exploring-data.html) and encountered two small issues which I worked around. I installed scippneutron by `pip install scippneutron[all]`  in a new pyenv on Ubuntu.

- When wanting to visualize instrument with `scn.instrument_view()` missing `pythreejs` dependency (installed it manually afterwards).
- Then got an error on the plotting:
image
which was solved by modifying `scene = fig.children[0].canvas.scene` to `scene = fig.canvas.scene` in `instrument_view.py`. 

Then it worked smoothly., but might as well report it.
Nice job btw.!

### PR DESCRIPTION
**Pull request by jl-wynen**
Fixes #508

### CODE REVIEW

**Comment by jl-wynen** [comment]
@jorobledo does this work for you?

File: docs/user-guide/installation.md [hunk 1]
```diff
@@ -12,12 +12,20 @@ pip install scippneutron
...
 ```sh
 pip install scippneutron[all]
 ```
 
+If you also want to use interactive plots in Jupyter notebooks, use
+
+```sh
+pip install scippneutron[interactive]
+# or
+pip install scippneutron[all, interactive]
```
Comment (nvaytet) [inline docs/user-guide/installation.md:26]: Isn't it kind of misleading that `all` doesn't bring everything in?
I think a user would assume `all` would just install all optional dependencies.
I know it comes from how it's done in Scipp, but maybe we should revisit there?

Comment (jl-wynen) [reply docs/user-guide/installation.md:26]: I agree. This is probably why it was written like this in the docs.

Comment (nvaytet) [reply docs/user-guide/installation.md:26]: I'll make a suggestion in a PR in scipp

Comment (jorobledo) [reply docs/user-guide/installation.md:26]: Thanks guys, crystal clear. But yeah, I would definitely expect that `all` brings up all that I need.

Maybe I would also add in the Tutorial documentation a small phrase stating something like "For an interactive experience on this tutorial install `pip install scippneutron[interactive]`"

Comment (nvaytet) [reply docs/user-guide/installation.md:26]: See https://github.com/scipp/scipp/pull/3433

Comment (nvaytet) [reply docs/user-guide/installation.md:26]: Now that https://github.com/scipp/scipp/pull/3433 is merged, we should just remove the `[interactive]` option in scippneutron and revert the other changes here.

Comment (jl-wynen) [reply docs/user-guide/installation.md]: Done

**Review by nvaytet — APPROVED** [review]

**Comment by nvaytet** [comment]
Maybe update the PR title? (which will make it into the release notes)

**Pull request merged**